src/tools/kg_tools.py

```
@@ -3,12 +3,17 @@
 """
 import os
 import json
+import logging
 from typing import Dict, Any
 
 from fastmcp import Context
 
+logger = logging.getLogger(__name__)
+
 from src.core.server import mcp
```

Ruff (0.12.2)

Module level import not at top of file

(E402)